containers/Connection.ts

```diff
@@ -132,14 +149,17 @@ function useConnection() {
       setMulticallAddress(42161, "0x813715eF627B01f4931d8C6F8D2459F26E19137E");
       setMulticallAddress(1285, "0x4c4a5d20f1ee40eaacb6a7787d20d16b7997363b");
       setMulticallAddress(25, "0x0fA4d452693F2f45D28c4EC4d20b236C4010dA74");
```

Comment:
https://api.pickle.finance/prod/protocol/pfcore inclueds a 'chain' element now, which also includes the multicall address.  If we can trust there'd be no man-in-the-middle attack, we could read this stuff from the pfcore json.  If we don't the security, we could still fetch it from the pfcore library instead.  Either way, it's not a good idea to duplicate this here.

Reply:
This is old code, will be removed. I just added the type for the code that's useful.